package.json

```diff
@@ -24,13 +24,14 @@
     "yarn": "^1.21.1"
   },
   "dependencies": {
+    "graphql-tag": "^2.11.0",
+    "graphql-tools": "^4.0.0",
     "lodash.difference": "^4.5.0",
-    "yargs": "^15.4.1"
+    "yargs": "^16.0.3"
```

Comment:
yargs should be devDependency, no?

Reply:
@barbieri The script used to generatype the typeMaps json uses yargs. This is why I left in dependencies.

Reply:
maybe peer?

Reply:
Not sure, at least the packages that I looked some the same strategy. Take a look for example in typeorm https://github.com/typeorm/typeorm/blob/master/package.json#L132
I think we should leave it as dependency
@barbieri

Reply:
ok, then